content/user/apps/TrackLibraries.md

```diff
@@ -32,22 +32,36 @@ Export details on findings from the Libraries page, a library's overview page, o
 
 <a href="assets/images/Library-export.png" rel="lightbox" title="Export library details"><img class="thumbnail" src="assets/images/Library-export.png"/></a>
 
-In the Libraries grids, select individual libraries with the check boxes or use your filters to focus on specific data sets. When you apply a selection, you can choose to export all or selected libraries. If you don't select any checkboxes or filters, you can export data for all libraries. 
+In the Libraries grids, select individual libraries or use the filters to focus on specific data sets. If you apply a selection, you can choose to export all or selected libraries; if not, you can export data for all libraries. 
 
 ### Data
 
-The exported file contains the following for each library:
-
-* NAME
-* TECHNOLOGY
-* RELEASE_DATE
-* VERSION
-* LATEST_VERSION
-* HASH
-* CVE <br> (A list of CVEs, separated by commas, found in this library.)
-* NUMBER_APPS <br> (The number of applications that use the library.)
-* NUMBER_CLASSES <br> (The number of classes that use this library.)
-
-
-
+The exported file contains the following data fields for each library:
+
+* Library Name
+* Language
+* Version
+* Release Date
+* Latest Version
+* Grade
+* SHA1
+* CVE Count
+* Application Count
+* Server Count
+* Number of Classes
+* Number of Used Classes
+
+You can access more details on your libraries by going to the [Contrast API documentation](https://api.contrastsecurity.com/#) **> Library > Libraries section**, or using the command line. 
+
+> **Example:** To find related application names and export the data to a CSV, use a cURL request to the API: 
```

Comment:
@gilday

Reply:
It's unfortunate that our API Docs do not allow us to generate permalinks to the exact section. We should consider adding this

Reply:
@gilday I will capture this in a ticket next week.

Reply:
"The following cURL request retrieves a list of libraries where each library includes a list of applications which use that library. The jq tool formats the data as CSV for use in a custom report"